src/test/java/org/broadinstitute/hellbender/GATKBaseTest.java

```diff
@@ -119,6 +123,16 @@ public void initGenomeLocParser() throws FileNotFoundException {
             locs.add(hg19GenomeLocParser.parseGenomeLoc(interval));
         return Collections.unmodifiableList(locs);
     }
-    
+
+    /**
+     * Closes the {@link java.nio.file.FileSystem} for a path if it is an instance from {@link Jimfs}
+     */
+    public static void closeFileSystemIfJimnfs(final Path path) throws IOException {
```

Comment:
I'm not sure this method should be up in `GATKBaseTest`, since you reported that you weren't able to use it in `CommandLineProgramIntegrationTest`. You should use the same method to create the Jimfs instances across your different test classes. At this point I'd be satisfied if you just deleted this method, and used the `@BeforeClass`/`@AfterClass` approach to set up the Jimfs in `IOUtilsUnitTest`.

Reply:
Deleted method.